Makefile

```diff
@@ -269,7 +272,7 @@ test-k6-performance: ## run k6 performance tests.
 
 .PHONY: web-build
 web-build: ng ## build the web components.
-	npm install --prefix web && npm run build --prefix web
+	npm install --prefix web && npm run build --prefix web --base-href $(WEB_ROOT)
```

Comment:
```suggestion
	npm install --prefix web && npm run build --prefix web -- --project yhs-components --base-href $(WEB_ROOT)
```
Hi, Current `web-build` command gives an error. Probably we need to change it to something I suggested above?

<img width="599" alt="image" src="https://github.com/user-attachments/assets/c0e18f20-164a-4bfc-89af-07779c875333">

Reply:
Ah, yes, npm requires `--` to propagate the flag to the script that is being executed. I have updated the PR